assets/js/components/setup/ModuleSetup.test.js

```diff
@@ -175,6 +180,20 @@ describe( 'ModuleSetup', () => {
 			it( 'should match the snapshot', () => {
 				expect( container ).toMatchSnapshot();
 			} );
+
+			it( 'tracks only the initial setup analytics view event', () => {
+				expect( mockTrackEvent ).toHaveBeenCalledWith(
+					`${ VIEW_CONTEXT_MODULE_SETUP }_setup`,
+					'setup_flow_v3_view_analytics_step'
+				);
+				const genericViewCall = mockTrackEvent.mock.calls.find(
+					( call ) =>
+						call[ 0 ] === 'moduleSetup' &&
+						call[ 1 ] === 'view_module_setup' &&
+						call[ 2 ] === MODULE_SLUG_ANALYTICS_4
+				);
+				expect( genericViewCall ).toBeUndefined();
```

Comment:
This test can be simplified:

```suggestion
				expect( mockTrackEvent ).toHaveBeenCalledTimes( 1 );
```